dev/build.changeDetector/src/com/ibm/ws/infra/depchain/ChangeDetector.java

```diff
@@ -424,12 +424,12 @@ private static void processOverallFeatureDeps(JsonObject fatMap, Map<String, Set
 
         // Convert: https://github.com/OpenLiberty/open-liberty/pull/5333
         // to:      https://github.com/OpenLiberty/open-liberty/pull/4947.diff
-        // to:      https://github.ibm.com/api/v3/repos/was-liberty/WS-CD-Open/pulls/13986/files
+        // to:      https://github.ibm.com/api/v3/repos/websphere/WS-CD-Open/pulls/13986/files
         String prNumber = prURL.substring(prURL.lastIndexOf("/"));
 
         // Decide which GitHub we need to make an API call to and choose the appropriate URL/access key
         String apiURL = prURL.contains("github.ibm") ? //
-                        "https://github.ibm.com/api/v3/repos/was-liberty/WS-CD-Open/pulls" + prNumber + "/files" : //
+                        "https://github.ibm.com/api/v3/repos/websphere/WS-CD-Open/pulls" + prNumber + "/files" : //
```

Comment:
Do we even need this logic?  Isn't it always open liberty here?  The GHE repo has its own copy of this code.

Reply:
Changed to only set the open-liberty apiURL.